instagram-app/src/components/PostContainer/postcontainer.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import './postcontainer.css';
+import Userinfo from './user/userinfo';
+import Postcontent from './postcontent/postcontent';
+import Useraction from './useractions/useraction';
```

Comment:
`import Useraction from './useractions/useractions';`

To match the file you have in your useractions folder.